config/mcus/can_ebb36.cfg

```diff
@@ -0,0 +1,28 @@
+[mcu toolhead]
+# This board work by using a serial connection by default.
+# If you want to use CAN, invert the commented lines here
+
+# serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+# canbus_uuid: change-me-to-the-correct-canbus-id
+
+
+[board_pins mcu_EBBCan]
```

Comment:
can you change the name here to `[board_pins mcu_ebb36_42]` ? This is to unify with the other canboard

Reply:
should we destinguish between v1.1 and v1.0

Reply:
Mhmm, I'm not familiar with these boards. Do you know if they have differences in the pinout ?

If it's the case, I would say yes, but if they are similar, I would say no.